scripts/get-estimated-service-fee.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from 'hardhat'
+import { CoordinatorBase__factory } from '@bisonai/orakl-contracts'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+async function main() {
+  const { vrfCoordinator: coordinatorAddress } = await hre.getNamedAccounts()
+  const coordinator = await ethers.getContractAt(CoordinatorBase__factory.abi, coordinatorAddress)
+
+  const reqCount = 1
+  const numSubmission = 1
+  const callbackGasLimit = 500_000
+  const estimatedGas = await coordinator.estimateFee(reqCount, numSubmission, callbackGasLimit)
```

Comment:
The variable name a bit confusing since it is not related to gas. How about changing it to `estimatedServiceFee`?